<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LCPNetworkToolsDialogBase</class>
 <widget class="QDialog" name="LCPNetworkToolsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LCP and Network Tools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLCP">
      <attribute name="title">
       <string>LCP Generation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_dem">
         <property name="text">
          <string>Input DEM Raster:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox_dem"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_start">
         <property name="text">
          <string>Start Point (X, Y):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_start">
         <item>
          <widget class="QLineEdit" name="lineEdit_start"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_selectStart">
           <property name="text">
            <string>Select on Map</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_dest">
         <property name="text">
          <string>Destination Points (X, Y):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_dest">
         <item>
          <widget class="QTextEdit" name="textEdit_dest">
           <property name="toolTip">
            <string>Enter one X,Y pair per line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_selectDest">
           <property name="text">
            <string>Select on Map</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_cost">
         <property name="text">
          <string>Cost Function:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_costMethod"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_move">
         <property name="text">
          <string>Move Type:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_moveType"/>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="checkBox_bothways">
         <property name="text">
          <string>Minimize for return journey (Both Ways)</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_usf">
         <property name="text">
          <string>Undersample Factor (usf):</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="spinBox_usf">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_osf">
         <property name="text">
          <string>Oversample Factor (osf):</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="spinBox_osf"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_corridor">
         <property name="text">
          <string>Corridor Width:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_corridor"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_maxtime">
         <property name="text">
          <string>Max Time (hours):</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_maxtime">
         <property name="maximum">
          <double>9999.990000000000</double>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_output_path">
         <property name="text">
          <string>Output Path Layer:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QgsFileWidget" name="mQgsFileWidget_outputPath"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_output_cost">
         <property name="text">
          <string>Output Cost Raster:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QgsFileWidget" name="mQgsFileWidget_outputCost"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNetwork">
      <attribute name="title">
       <string>Network Generation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_nodes">
         <property name="text">
          <string>Input Nodes Layer:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox_nodes"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_dem_net">
         <property name="text">
          <string>Input DEM Raster:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox_dem_net"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_lambda">
         <property name="text">
          <string>Lambda (Î»):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_lambda">
         <property name="value">
          <double>7.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_maxrad">
         <property name="text">
          <string>Max Radius (hours):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_maxrad">
         <property name="value">
          <double>36.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_algo">
         <property name="text">
          <string>Algorithm:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_algo"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_output_net">
         <property name="text">
          <string>Output Network Layer:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QgsFileWidget" name="mQgsFileWidget_outputNet"/>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_log">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>LCPNetworkToolsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>LCPNetworkToolsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
